pyproject.toml

```diff
@@ -20,6 +20,7 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.9"
 linkml-runtime = "^1.6.2"
+gspread = "^6.1.4"
```

Comment:
shall we remove this dependency?

Reply:
All of the notebooks in `notebooks/environmental_context_value_sets/` use `gspread`.

I'll move it to `tool.poetry.group.dev.dependencies`